runtime/parser/schema/project.schema.yaml

```diff
@@ -1240,13 +1243,51 @@ definitions:
                   type: string
                   description: used to configure what value to fill in for missing time buckets. This also works generally as COALESCING over non empty time buckets.
             minItems: 1
+          parent_dimensions:
+            description: Optional field selectors for dimensions to inherit from the parent metrics view.
+            $ref: '#/definitions/fields_selector'
+          parent_measures:
+            description: Optional field selectors for measures to inherit from the parent metrics view.
+            $ref: '#/definitions/fields_selector'
           security:
             $ref: '#/definitions/security_policy_properties'
             description: Defines a security policy for the dashboard
+          explore:
+            $ref: '#/definitions/explore_properties'
+            description: Defines an inline explore view for the metrics view
         required:
           - type
       - $ref: '#/definitions/common_properties' 
     definitions:
+      fields_selector:
```

Comment:
Does this duplicate `field_selector_properties` from line 1057? Should it perhaps be pulled out as a shared type instead?